### PR TITLE
[INFRA-1690] overrides workflow-support with tag

### PR DESCRIPTION
**Pull request by MRamonLeon**
See **[[INFRA-1690] DevTools: Add support of Specifying Java Level for Test in runATH()
](https://issues.jenkins-ci.org/browse/INFRA-1690)**

Overrides the workflow-support plugin, add the modules/libraries removed from java 11 and do that only on java 11.

Block by #464 (jdk11 in the docker image)

@olivergondza @jenkinsci/java11-support @raul-arabaolaza

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -20,13 +20,19 @@ for (int j in [8, 11]) {
                         def exclusions = splits.get(index).join("\n")
                         writeFile file: 'excludes.txt', text: exclusions
                         realtimeJUnit(testResults: 'target/surefire-reports/TEST-*.xml', testDataPublishers: [[$class: 'AttachmentPublisher']]) {
-                            sh '''
-                                eval $(./vnc.sh)
+                            def java11mods = (javaVersion == 11) ? '''
                                 # Temporary to get Java 11 going: https://github.com/jenkinsci/workflow-support-plugin/pull/68#issuecomment-440971292
```
Comment (oleg-nenashev) [inline Jenkinsfile]: Comment probably needs an update as well (e.g. issue reference: https://issues.jenkins-ci.org/browse/JENKINS-51998)

**Comment by oleg-nenashev** [comment]
IIUC it will not behave correctly until https://github.com/jenkinsci/acceptance-test-harness/pull/464 is integrated, right?

**Comment by MRamonLeon** [comment]
It's independent. Only the https://github.com/jenkins-infra/pipeline-library/pull/82 is dependent of #464 (docker image for pipeline steps)

**Comment by MRamonLeon** [comment]
Sorry Oleg. It has no sense to have this merged without merging the #464 that introduces the jdk11 in the docker image. So you're right.

**Comment by olivergondza** [comment]
Merging then. Thank you!

**Pull request merged**